### PR TITLE
Remove archived integration [custom-components/sensor.rpi_power]

### PR DESCRIPTION
**Pull request by hmmbob**
Remove the custom-components/sensor.rpi_power integration as the repo has been archived

### CODE REVIEW

**Comment by ludeeus** [comment]
This is not enough, run `make remove` <https://hacs.xyz/docs/publish/remove>

**Review by haberda — APPROVED** [review]

**Comment by hmmbob** [comment]
The script errors out, unfortunately:

```bash
bob@debian:~/hacs/default$ make remove
Repository: custom-components/sensor.rpi_power
Reason: Repo is archived
Removal type: integration
Link to more information: https://github.com/custom-components/sensor.rpi_power
custom-components/sensor.rpi_power
Repo is archived
integration
https://github.com/custom-components/sensor.rpi_power
  File "scripts/remove_repo.py", line 56
    print(f"Found in {category}")
                               ^
SyntaxError: invalid syntax
bob@debian:~/hacs/default$
```

**Comment by ludeeus** [comment]
It needs python 3.7, if you don't have that this repo have a devcontainer that provides it, and it's also compatible with github codespaces.

**Comment by hmmbob** [comment]
Tx @ludeeus. It appears that there is one `python` vs `python3` mention in the `make remove` flow, when changed to `python3` it works flawlessly. I'll put in a PR so you can see if it is worth changing.

Think this should be good now, let me know if not.

File: removed [hunk 1]
```diff
@@ -411,5 +411,11 @@
     "reason": "Deprecated",
     "removal_type": "remove",
     "link": "https://github.com/custom-components/uilogs/issues/17"
+  },
+  {
+    "repository": "custom-components/sensor.rpi_power",
+    "reason": "Repo archived",
+    "removal_type": "integration",
```
Comment (ludeeus) [inline removed:418]:
```suggestion
    "removal_type": "removal",
```

**Review by ludeeus — APPROVED** [review]

**Comment by hmmbob** [comment]
Thanks @ludeeus!

**Pull request merged**